### PR TITLE
Change default config

### PR DESCRIPTION
**Pull request by nikooo777**
It's very confusing when it comes to changing the variables.
I never had a service where DEFAULT- was prepended and I had to figure out that it had to be removed in order to set the value.

My suggestion is to simplify it as such

### CODE REVIEW

**Comment by nikooo777** [comment]
example of a PHP.ini configuration:

```
; Maximum amount of time each script may spend parsing request data. It's a good
; idea to limit this time on productions servers in order to eliminate unexpectedly
; long running scripts.
; Note: This directive is hardcoded to -1 for the CLI SAPI
; Default Value: -1 (Unlimited)
; Development Value: 60 (60 seconds)
; Production Value: 60 (60 seconds)
; http://php.net/max-input-time
;max_input_time = 
```

**Comment by tiger5226** [comment]
@Leopere I think this PR will require lbry-docker changes. Can you confirm or deny?

**Comment by Leopere** [comment]
It will but not much, I set up the configurator script to be pretty easy to modify.  

https://github.com/lbryio/lbry-docker/blob/leopere-nov-11/chainquery/stuff/start.sh#L5-L15

This just needs line 12-15 updated to match the new patterns.

**Comment by nikooo777** [comment]
I updated my suggestion to a better version. Please see if that is still in line with your parser

**Comment by nikooo777** [comment]
the only reason one would comment out the var is to set it to something that isn't default, so I disagree that we should leave the default in the commented out var. I've looked around and several other projects do the same.

perhaps @tiger5226 can pitch in?

File: config/default/chainqueryconfig.toml [hunk 1]
```diff
@@ -1,60 +1,84 @@
...
-#DEFAULT-twiliosid=""
-#DEFAULT-twilioauthtoken=""
-#DEFAULT-smsrecipients=["",""]
-#DEFAULT-smsfromphonenumber=""
\ No newline at end of file
+#DEFAULT:
+##twiliosid=""
+##twilioauthtoken=""
+##smsrecipients=["",""]
+##smsfromphonenumber=""
```
Comment (nikooo777) [inline config/default/chainqueryconfig.toml]: i'm skeptical of the 4 lines above. perhaps each var should have its own description and default value to remain consistent

Comment (tiger5226) [reply config/default/chainqueryconfig.toml]: I think it's fine. They are a group of related parameters. We can add more of a description if needed, but I don't think we should separate them. Maybe we can group them into a toml object instead? Not sure that would be a valuable effort.

**Pull request merged**